### PR TITLE
Add expenses Ui and list expenses command 

### PR DESCRIPTION
**Pull request by Jingwencloud**

### CODE REVIEW

File: src/main/java/seedu/awe/logic/parser/AddressBookParser.java [hunk 1]
```diff
@@ -1,5 +1,4 @@
 package seedu.awe.logic.parser;
-
```
Comment (marcuspeh) [inline src/main/java/seedu/awe/logic/parser/AddressBookParser.java:2]: Should there be a line spacing here?

File: src/main/java/seedu/awe/model/ReadOnlyAddressBook.java [hunk 2]
```diff
@@ -1,10 +1,12 @@
 package seedu.awe.model;
 
 import javafx.collections.ObservableList;
+import seedu.awe.model.expense.Expense;
 import seedu.awe.model.group.Group;
 import seedu.awe.model.group.GroupName;
 import seedu.awe.model.person.Person;
 
+
```
Comment (marcuspeh) [inline src/main/java/seedu/awe/model/ReadOnlyAddressBook.java:9]: Is it better if there is only one line spacing?

File: src/main/java/seedu/awe/model/group/Group.java [hunk 3]
```diff
@@ -113,6 +113,7 @@ public boolean isPartOfGroup(Person person) {
     }
 
     public ArrayList<Expense> getExpenses() {
+
```
Comment (marcuspeh) [inline src/main/java/seedu/awe/model/group/Group.java:116]: Perhaps this line spacing can be removed?

File: src/main/java/seedu/awe/ui/CommandBox.java [hunk 4]
```diff
@@ -49,6 +49,7 @@ private void handleCommandEntered() {
         }
     }
 
+
```
Comment (marcuspeh) [inline src/main/java/seedu/awe/ui/CommandBox.java:52]: Perhaps it can be removed too

File: src/main/java/seedu/awe/ui/MainWindow.java [hunk 5]
```diff
@@ -18,6 +18,7 @@
 import seedu.awe.logic.parser.exceptions.ParseException;
 
 
+
```
Comment (marcuspeh) [inline src/main/java/seedu/awe/ui/MainWindow.java:21]: I think there should only be 1 line spacing here

**Review by kheekheekhee — APPROVED** [review]
LGTM! 👍

**Pull request merged**